app/assets/stylesheets/views/_travel-advice.scss

```diff
@@ -1,19 +1,14 @@
 .travel-advice {
   @include parts;
-
-  .add-title-margin {
-    @include responsive-top-margin;
-  }
+  @include add-title-margin;
 
   .part-navigation {
     margin-bottom: $gutter * 1.5;
   }
 
-  .map {
+  // TODO: Remove this when components can accept variable unidirectional spacing,
+  // the component above should have a bigger margin-bottom.
+  .app-c-figure {
     margin-top: $gutter;
```

Comment:
I think a stray margin-top is safer than overriding a component. TODO statement would still apply.

Reply:
Resolved